packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -118,27 +126,28 @@
 .rt-RadioCardGroupRoot:where(.rt-variant-classic) {
   :where(.rt-RadioCardGroupItem) {
     background-color: var(--color-surface);
-    /* A transparent shadow to transition smoothly to `:active` */
-    box-shadow: 0 0 0 1px var(--color-transparent), var(--shadow-2);
+    box-shadow: var(--shadow-2);
     transition: box-shadow 120ms;
 
-    &:where(:not([data-state='checked'])) {
-      @media (hover: hover) {
-        &:where(:hover) {
-          transition-duration: 40ms;
-          box-shadow: var(--card-classic-hover-box-shadow);
-        }
-      }
-
-      &:where(:active) {
+    @media (hover: hover) {
+      &:where(:not(:disabled):not([data-state='checked']):hover) {
         transition-duration: 40ms;
-        box-shadow: 0 0 0 1px var(--gray-a5), var(--shadow-2);
+        box-shadow: var(--card-classic-hover-box-shadow);
       }
     }
 
+    &:where(:disabled) {
+      background-color: var(--gray-a2);
+      color: var(--gray-a9);
+    }
+
     &:where([data-state='checked']) {
       outline: 2px solid var(--accent-9);
       outline-offset: -1px;
+
+      &:where(:disabled) {
+        outline-color: var(--gray-8);
+      }
```

Comment:
We need to make sure this also works for high contrast

<img width="422" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/845cd977-452c-4e7c-859d-d9c2221dd855">

Reply:
Yep